src/Illuminate/Cache/ArrayStore.php

```diff
@@ -184,7 +184,7 @@ protected function calculateExpiration($seconds)
      * Get the UNIX timestamp for the given number of seconds.
      *
      * @param  int  $seconds
-     * @return int
+     * @return float
```

Comment:
This is a breaking change

Reply:
https://github.com/laravel/framework/pull/48582#discussion_r1341011023